pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor.
+
+- Compress file:
+
+`bzip2 {{path/to/file_to_compress}}`
+
+- Decompress file:
+
+`bzip2 -d {{path/to/compressed_file.bz2}}`
+
+- Decompress to STDOUT:
+
+`bzip2 -dc {{path/to/compressed_file.bz2}}`
```

Comment:
Man, I wish there was a word starting in "c" related to stdout, piping, or something like that, which we could use to make the short option easier to memorize...